pkg/network/openshift_sdn.go

```diff
@@ -123,8 +125,15 @@ func fillOpenShiftSDNDefaults(conf *netv1.NetworkConfigSpec) {
 		var port uint32 = 4789
 		sc.VXLANPort = &port
 	}
+
+	// MTU is currently the only field we pull from previous.
+	// If it's not supplied, we infer it from  the node on which we're running.
+	// However, this can never change, so we always prefer previous.
 	if sc.MTU == nil {
-		var mtu uint32 = 1450
+		var mtu uint32 = uint32(hostMTU) - 50 // 50 byte overhead
```

Comment:
Nit: Can you say '50 byte overhead for VxLAN header' or something

Reply:
Nit nit: "VXLAN"